src/components/Filter/FilterInput.vue

```diff
@@ -56,36 +56,36 @@
           <label
             id="filter-label"
             :for="FilterInputId"
-            class="visuallyhidden"
-            aria-hidden="true"
+            :input-value="modelValue"
```

Comment:
I would suggest the same, as I did for @hqhhuang. Use a `data-` attribute, so note its a custom data attribute. Now it looks like a vue prop.

Reply:
https://github.com/apple/swift-docc-render/pull/291/commits/34c3848b2287a7b62d7639a4bd1ff8df5614ec9a